templates/ampache.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>ampache</Name>
+  <Repository>ampache/ampache</Repository>
+  <Registry>https://hub.docker.com/r/ampache/ampache/</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/ampache/ampache/issues</Support>
+  <Project>https://ampache.org/</Project>
```

Comment:
Add a note that users should follow configuration steps on the project's wiki once the container is running: https://github.com/ampache/ampache/wiki/Basic

Reply:
added commit including this.